MainTel/MainTel-Troubleshooting/MTCS-Troubleshooting/TMA-troubleshooting/TMA-Troubleshooting-OSS-Fails-to-Turn-On-index.rst

```diff
@@ -0,0 +1,181 @@
+.. _Templates-Title-of-Troubleshooting-Procedure:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+########################
+TMA OSS Fails to Turn On
+########################
+
+
+.. _Title-of-Troubleshooting-Procedure-Overview:
+
+Overview
+========
+
+.. In one or two sentences, explain when this troubleshooting procedure needs to be used. Describe the symptoms that the user sees to use this procedure. 
+
+This procedure describes how to recover the OSS system when it fails to turn ON. 
+.. _Title-of-Troubleshooting-Procedure-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+.. This section should provide simple overview of known or suspected causes for the error.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Post screenshots of the error state or relevant tracebacks.
+
+- Chiller(s) light in the EUI OSS screen is red. **"OSS"**, **"observation"** and **"cooling"** lights are red too. On the alarm banner on the bottom, you can read **"LOCAL: COOLING SYSTEM"**
+
+.. figure:: ./_static/OSS_general_view.png
+   :name: OSS general view 
+
+
+
+
+
+
+The **Alarm History** screen shows
+
+.. figure:: ./_static/alarm_history.png
+    :name:  Alarm History
+
+.. _Title-of-Troubleshooting-Procedure-Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. todo::
+   Make sure everything is in a safe or idle state before troubleshooting. Describe relevant safety steps if necessary.
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+
+
+
+.. _Title-of-Troubleshooting-Procedure-Critical-Step-1:
+
+ 1. Go to Level 1 Machinery Room and bring a Philips screwdriver with you. The Machinery Room is in the left door in the picture below.
+
+
+
+
+.. figure:: ./_static/level-1.png
+    :name: level 1
+
+
+
+
+2. The screens in the front panel of one or both of the chiller cabinets would display an alarm. On the top, the red band may say: **“STATUS: NOT ACKNOWLEDGED”** or **“STATUS: ALARM”**
```

Comment:
I think here it´s better to remove the conditional "would", and also the "may" in "may say"